commands/orchestrationcommands/buildinfocommands/buildinfo.go

```diff
@@ -0,0 +1,69 @@
+package buildinfocommands
+
+import (
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	"github.com/rackspace/rack/internal/github.com/fatih/structs"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/buildinfo"
+	"github.com/rackspace/rack/util"
+)
+
+var commandPrefix = "orchestration"
+
+var BuildInfo = cli.Command{
+	Name:        "buildinfo",
```

Comment:
But that's the point: All `rack` commands that interact with the Rackspace API are of the following form:
`rack <service> <sub-service> <action>`
Consistency is one of the top tenets of this tool.